Small design changes

D0nPiano opened the pull request:
See commits for changes

tiltec commented on src/pages/Store/Pickups.vue:
```diff
@@ -26,8 +26,15 @@
+              <span class="fa-stack fa-lg">
+                <i
+                  class="fa fa-shopping-basket
+                  fa-stack-2x bottom-icon"/>
```
Better no linebreak inside the class attribute, looks like two attributes.

tiltec commented on src/pages/Store/Pickups.vue:
```diff
@@ -47,12 +54,21 @@
+            <span class="fa-stack fa-lg">
+              <i
+                class="fa fa-shopping-basket
+                fa-stack-2x bottom-icon"/>
```
same here: no linebreak would be better

D0nPiano commented:
Thanks for the feedback! - Understandable, changed them to calendar icons and also added the memberlist to groupOptions.

Will leave the PR for now. Awaiting review.
